src/main/java/com/ericsson/ei/controller/model/ParseInstanceInfoEI.java

```diff
@@ -137,16 +139,41 @@ public void init() throws IOException {
     @Component
     private class MongoDbValues {
         @Getter
-        @Value("${spring.data.mongodb.host}")
-        private String host;
-
-        @Getter
-        @Value("${spring.data.mongodb.port}")
-        private String port;
+        @Value("${spring.data.mongodb.uri}")
+        private String uri;
 
         @Getter
         @Value("${spring.data.mongodb.database}")
         private String database;
+
+        @PostConstruct
+        public void init() throws IOException {
+            if (uri != null) {
+                String secret = extractSecretFromUri(uri);
+                String modifiedUri = uri.replaceFirst(secret, "hidden_secret");
+                uri = modifiedUri;
+            }
+        }
+
+        /**
+         * This method should be moved to decrypt library once EI 2.0.2 is merget to master
+         *
+         * @param uri
+         * @return
+         */
+        private String extractSecretFromUri(String uri) {
+            String secret = "";
+            // Matcher that match string between : and @
```

Comment:
Maybe give an example of what the entire string looks like as well. ": and %" doesnt say much when you don't know how the uri looks like :)

Reply:
Will do better, code moved to iuts own class.

Reply:
Done in next commit.